Shared/UserAgent.swift

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Remember I said to watch out for Xcode breaking the indentation here? All of the '*' at the start of the block should be in one vertical line.